README.md

```diff
@@ -1,3 +1,107 @@
 # Intel NPU Driver Snap
 
-Snap recipe for the [Intel NPU Driver](https://github.com/intel/linux-npu-driver/). This snap is designed to be a producer snap providing NPU (neural processing unit) firmware and user-space libraries (including the user mode driver and NPU compiler) for consumption by application snaps via the [content interface](https://snapcraft.io/docs/content-interface).
+Snap recipe for the [Intel NPU Driver](https://github.com/intel/linux-npu-driver/). This snap is designed to be a producer snap providing NPU (neural processing unit) firmware, char device node access, and user-space libraries (including the user mode driver and NPU compiler) for consumption by application snaps. It exposes slots for consumer snaps to connect to (see below) but also provides firmware binary blobs for the NPU device and packages an app for validating the user space driver (`vpu-umd-test`).
```

Comment:
is the term `producer snap` coming from any existing documentation ? i m seeing that some existing similar snaps are characterized as `provider snap`

Reply:
The terms producer and consumer are used throughout the [content interface docs](https://snapcraft.io/docs/content-interface). For example: 

> The content interface allows sharing code and data from a slot on a producer snap to one or more plugs on consumer snaps. It’s often used to share common themes and icons from a producer snap, for example, to many snapped applications. Sharing happens at the filesystem level, which means anything that can be expressed as a file can be shared. This includes executables, libraries and data files, but also sockets.

I looked around briefly for `provider snap` but didn't see much. Maybe used interchangeably with `producer snap`, or do you think it's something different completely?

Reply:
yeah, both producer and provider terms are used in the documentation, so that is fine if you use producer